RecordCollection/Controllers/AlbumsController.cs

```diff
@@ -15,8 +19,39 @@ public AlbumsController(RecordCollectionContext context)
         public IActionResult Index()
         {
             var albums = _context.Albums.ToList();
-            
             return View(albums);
         }
+
+        [Route("Albums/{id:int}")]
+        public async Task<IActionResult> Show(int? id)
+        {
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var retrieved = await _context.Albums.FindAsync(id);
+
+            var relatedAlbums = _context.Albums
+                .Where(a => a.Artist == retrieved.Artist && a.Id != retrieved.Id)
+                .OrderBy(a => a.ReleaseDate);
+
+            var viewModel = new AlbumViewModel
```

Comment:
Let's update the name for this viewModel variable. I believe it could become confusing for new eyes especially as the program grows. Could we name it to something that more clearly illustrates that it is an AlbumViewModel?